### ISSUE TITLE
css : 404 on default bundle

### ISSUE THREAD

**Issue by ArthurJam** [Kunstmaan/KunstmaanBundlesCMS#1356]
Hello,

I dont understand how i can edit the css files and how i can load them.
By default, i have a 404 error from frontend/css/style.min.css

Do i missed something ?
Thanks.

**Comment by ArthurJam** [comment]
I found this workaround : 

In views\Layout\_css.html.twig i change the content by this : 

```
{% block css %}
    {% stylesheets "@MyWebsiteBundle/Resources/ui/scss/style.scss" output="MyWebsiteBundleAssets/css/style.min.css" filter="scss" %}
    <link href="{{ asset_url }}" type="text/css" rel="stylesheet" media="screen"/>
    {% endstylesheets %}
{% endblock %}
```

In app/config/config.yml in assetic section  : 

```
assetic:
    bundles: [MyWebsiteBundle]
    filters:
        scss:
            load_paths:
                - "%kernel.root_dir%/.."
```

The execute a cache:clear and assetic:dump

I think i misunderstood something (including how to write in english 😄 ).

**Comment by Devolicious** [comment]
@ArthurJam Are you talking about the frontend css? If so, this is done with Gulp. By generating a default site with the `bin/console kuma:generate:default-site` command a gulpfile will be generated. The whole process to build the frontend is explained here: https://bundles.kunstmaan.be/documentation/master/installation

**Comment by giovannilegrandpp** [comment]
@Devolicious i have used that tutorial but it ends up with: npm run build this gives me the message that the script build is missing, i have manually build it with gulp but still no proper theming, could you help me further with this?

**Comment by Devolicious** [comment]
Hi @giovannilegrandpp 

Since my last answer there has been a change in our cms and the documentation. I presume that you installed the 4.0.2 release. The correct documentation for this is under https://bundles.kunstmaan.be/documentation/4.0/installation

We are currently reworking our frontend tools and there is already a part of that available, if you start from `dev-master`. The `npm run build` is part of this rework and is not available in the 4.0.2 release.

Hope this helps.
